mrmustard/lab_dev/states/base.py

```diff
@@ -236,6 +237,26 @@ def from_fock(
                 modes.
         """
 
+    @classmethod
+    @abstractmethod
+    def from_modes(
```

Comment:
out of curiosity, why not `from_ansatz`?

Reply:
Glad you mention that because I was actually considering renaming to that 😁